book/src/rent.md

```diff
@@ -0,0 +1,52 @@
+# Rent
+
+Accounts on Solana may have owner-controlled state (`Account::data`) that's separate 
+from the account's balance (`Account::lamports`).  Since validators on the network need 
+to maintain a working copy of this state in memory, the network charges a time-and-space 
+based fee for this resource consumption, also known as Rent.
+
+## Two-tiered rent regime
+
+Accounts which maintain a minimum balance equivalent to 2 years of rent are exempt from
+rent.  Accounts whose balance falls below this threshold are charged rent 
+at a rate specified in genesis, in lamports per kilobyte-year.
+
+## Collecting rent
+
+Rent is charged at the end of each slot on all accounts that were accessed during that block.  A field in the account `Account::rent_slot` is keeps track of the last time rent was paid by this account.
```

Comment:
“rent_block” right?

Reply:
slot is what I would prefer, because that is more like "time" to me